### ISSUE TITLE
⛔ ♿ 🐞 Bug Report: Elementor Map Widget not accessible WCAG 2.1 AA

### ISSUE THREAD

**Issue by alfredo49** [elementor/elementor#20659]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The Axe DevTools return this error:

"Use aria-roledescription on elements with a semantic role"

**Issue Description**
Ensure aria-roledescription is only used on elements with an implicit or explicit role

**Element Location**
`iframe #mapDiv > div > .gm-style > div[aria-label="Mapa"][aria-roledescription="mapa"][role="region"]`

**Element Source**
`<div tabindex="0" aria-label="Mapa" aria-roledescription="mapa" role="region" style="position: absolute; z-index: 0; left: 0px; top: 0px; height: 100%; width: 100%; padding: 0px; border-width: 0px; margin: 0px; cursor: url(&quot;https://maps.gstatic.com/mapfiles/openhand_8_8.cur&quot;), default; touch-action: pan-x pan-y;" aria-describedby="A38722FD-D937-4F8D-ADA2-D888834774E4">`

To solve this problem, you need to...

Fix the following:
Check that the aria-roledescription is announced by supported screen readers

### Steps to reproduce

Insert a google map widget and test accesibility for example with Axe Dev Tools Widget Chrome

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.5.16
	PHP Version: 8.0.23
	PHP Memory Limit: 
	PHP Max Input Vars: 10000
	PHP Max Post Size: 35M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://www.tqmdesarrollo.com/kitdigital
	Home URL: https://www.tqmdesarrollo.com/kitdigital
	WP Multisite: No
	Max Upload Size: 35 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: es
	Timezone: Europe/Madrid
	Admin Email: alfredo@trisquel.com
	Debug Mode: Inactive

== Theme ==
	Name: Astra
	Version: 3.9.4
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: es_ES
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:107.0) Gecko/20100101 Firefox/107.0

== Active Plugins ==
	Code Snippets
		Version: 3.2.2
		Author: Code Snippets Pro

	Elementor
		Version: 3.9.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

	Rank Math SEO
		Version: 1.0.103.1
		Author: Rank Math

	Really Simple SSL
		Version: 6.0.12
		Author: Really Simple Plugins

== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

== Uso de los elementos ==
	
	container : 0
		container : 4
		image : 1
		social-icons : 1
	footer : 1
		container : 4
		html : 1
		image : 1
		social-icons : 1
		theme-site-logo : 1
	header : 1
		container : 1
		nav-menu : 1
		theme-site-logo : 1
	wp-page : 1
		button : 2
		container : 21
		form : 1
		google_maps : 1
		heading : 5
		icon : 1
		image : 4
		lottie : 1
		text-editor : 6

== Experimentos de Elementor ==
	Salida del DOM optimizada: Activo
	Carga de recursos mejorada: Activo
	Carga de CSS mejorada: Activo
	Fuentes de iconos integradas: Inactivo por defecto
	Mejoras de accesibilidad: Activo
	Puntos de ruptura personalizados adicionales: Inactivo por defecto
	Importar o exportar un kit web: Activo por defecto
	Ocultar los widgets nativos de WordPress de los resultados de búsqueda: Activo por defecto
	admin_menu_rearrangement: Inactivo por defecto
	Contenedor Flexbox: Activo
	Por defecto, el nuevo constructor de temas: Activo por defecto
	Páginas de destino: Activo por defecto
	Muestrario de colores: Activo por defecto
	Widgets favoritos: Activo por defecto
	Barra superior de administración: Activo por defecto
	Carga diferida de imágenes de fondo: Inactivo por defecto
	Guardar como por defecto: Activo por defecto
	Transiciones de páginas: Inactivo
	Notes: Inactivo
	Bucle: Inactivo
	Envíos de formulario: Activo por defecto
	Ajuste de scroll: Inactivo

== Registro ==
	
Log: showing 20 of 402022-12-05 11:22:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.6',
  'to' => '3.8.1',
)]
2022-12-05 11:22:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.6',
  'to' => '3.8.1',
)]
2022-12-05 11:22:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.6',
  'to' => '3.8.1',
)]
2022-12-05 11:25:44 [info] elementor-pro::elementor_pro_updater Started 
2022-12-05 11:25:44 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-05 11:25:44 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-05 11:25:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.2',
  'to' => '3.8.2',
)]
2022-12-05 11:25:44 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.2',
  'to' => '3.8.2',
)]
2022-12-07 08:30:39 [info] elementor-pro::elementor_pro_updater Started 
2022-12-07 08:30:39 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-07 08:30:39 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-07 08:30:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 08:30:39 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 08:30:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 08:30:48 [info] elementor::elementor_updater Started 
2022-12-07 08:30:48 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-07 08:30:48 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-07 08:30:48 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-12-07 08:30:48 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-12-07 08:30:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]

JS: showing 1 of 1JS: 2022-12-05 10:44:42 [error X 11][http://www.tqmdesarrollo.com/kitdigital/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.8.1:2:30926] elementor.settings.page is undefined 

PHP: showing 1 of 1PHP: 2022-12-05 11:47:58 [warning X 13][/usr/home/tqmdesarrollo.com/web/kitdigital/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

**Comment by rami-elementor** [comment]
HI @alfredo49 

Elementor **Map Widget** adds only the `<iframe>` tag as follows:

```html
<iframe frameborder="0" scrolling="no" marginheight="0" marginwidth="0"
src="https://maps.google.com/maps?q=London%20Eye%2C%20London%2C%20United%20Kingdom&amp;t=m&amp;z=10&amp;output=embed&amp;iwloc=near"
title="London Eye, London, United Kingdom" aria-label="London Eye, London, United Kingdom">
</iframe>
```

You mentioned that using "Axe DevTools" you see an error in the following element:

> **Element Location**
> `iframe #mapDiv > div > .gm-style > div[aria-label="Mapa"][aria-roledescription="mapa"][role="region"]`

It's not an Elementor issue. It look like a Google Map error. Elementor does not adds inner elements to the `iframe` tag.

The element with the `#mapDiv` id is an element inside the iframe, which we can't change.
